cogs/filters.py

```diff
@@ -64,6 +64,45 @@ async def bulk_load_config(self, ctx):
         else:
             await ctx.send("There is no valid file for loading!")
 
+    @is_staff("Helper")
+    @commands.group()
+    async def invitefilter(self, ctx):
+        """Command group for managing the invite filter"""
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help(ctx.command)
+
+    @is_staff("SuperOP")
+    @invitefilter.command(name='add')
+    async def add_invite(self, ctx, invite: discord.Invite, alias: str):
+        """Adds a invite to the filter whitelist"""
+        if await self.bot.invitefilter.fetch(code=invite.code, alias=alias, separator='OR'):
+            return await ctx.send("This invite code or alias is already in use!")
+        name, _ = await self.bot.invitefilter.add(name=invite.guild.name, code=invite.code, alias=alias, uses=-1)
+        if name is None:
+            return await ctx.send("Failed to add invite to the invite whitelist!")
+        await self.bot.channels['mod-logs'].send(f"🆕 **Added**: {ctx.author.mention} added {invite.code}(`{invite.guild.name}`) to the invite whitelist!")
+        await ctx.send("Successfully added invite to whitelist")
+
+    @is_staff("SuperOP")
+    @invitefilter.command(name='delete')
+    async def delete_invite(self, ctx, code: str):
+        """Removes a invite from the filter whitelist"""
+        name, _ = await self.bot.invitefilter.delete(code=code)
```

Comment:
This will always fail with `TypeError: cannot unpack non-iterable NoneType object` because invitefilter.delete doesn't return anything.